### PR TITLE
Enable Google Analytics for prow.k8s.io

### PR DESCRIPTION
**Pull request by Katharine**

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ deck:
   tide_update_period: 1s
   hidden_repos:
   - kubernetes-security
+  google_analytics: UA-82843984-5
```
Comment (cblecker) [inline prow/config.yaml]: is this the same analytics account used for the rest of our stuff? cc: @castrojo

Comment (Katharine) [reply prow/config.yaml]: It's the same as the rest of the sig-testing stuff.

**Pull request merged**